linkerd/protocol/http/src/main/scala/com/twitter/finagle/buoyant/linkerd/LinkerdHeaders.scala

```diff
@@ -49,6 +52,19 @@ import scala.collection.breakOut
 object Headers {
   val Prefix = "l5d-"
 
+  /**
+   * A param specific to error responses, the maximum size of a HTTP error response payload.
+   * By default it has the same value as maxResponseKB. Example:
+   * - protocol: http
+   *   maxResponseKB: 5120
+   */
+  object param {
+    case class MaxErrResponseSize(size: StorageUnit)
```

Comment:
TIOLI: Its uasually good practice to have your case classes being `final`. Can read more about it [here](https://nrinaudo.github.io/scala-best-practices/tricky_behaviours/final_case_classes.html). Also you can have the case class extending `AnyVal` to avoid the extra allocation (not that it will matter much)

Reply:
done